### DISCUSSION TITLE
Wazuh rules protecting log data?

### DISCUSSION THREAD

**Discussion by juddcbrown** [Security-Onion-Solutions/securityonion#5035] [Unsupported Versions] [answered]
Is anyone familiar with a Wazuh/OSSEC rule that monitors the log data itself on the Search node? PCI question.

I know FIM is meant for more static files - any input is appreciated.

Regards.

**Comment by jctello** [comment, marked as answer]
Hi @juddcbrown , 
Given that the data is stored on both the original log file, the wazuh alerts log and the Elasticsearch indices immediately after the event is generated then there are several mechanisms by which data tampering would be evident.
On the search node there will be logs for API calls regarding deletion events. Also if any file is unexpectedly reduced in size when being monitored by the Wazuh agent then [rule 592](https://github.com/wazuh/wazuh/blob/v4.3.5/ruleset/rules/0015-ossec_rules.xml#L293-L301) will be triggered.

**Comment by juddcbrown** [comment]
Thank you Juan!


On Mon, Jul 18, 2022 at 02:31 Juan Carlos Tello ***@***.***>
wrote:

> Hi @juddcbrown <https://github.com/juddcbrown> ,
> Given that the data is stored on both the original log file, the wazuh
> alerts log and the Elasticsearch indices immediately after the event is
> generated then there are several mechanisms by which data tampering would
> be evident.
> On the search node there will be logs for API calls regarding deletion
> events. Also if any file is unexpectedly reduced in size when being
> monitored by the Wazuh agent then rule 592
> <https://github.com/wazuh/wazuh/blob/v4.3.5/ruleset/rules/0015-ossec_rules.xml#L293-L301>
> will be triggered.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/5035#discussioncomment-3169093>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH4XT7UFMWI3KZKZBTU44NLVUUB37ANCNFSM5BNRJFPA>
> .
> You are receiving this because you were mentioned.Message ID:
> <Security-Onion-Solutions/securityonion/repo-discussions/5035/comments/3169093
> @github.com>
>
-- 
Judd Brown
